tools/metrics/import/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '3.8'
+
+networks:
+  monitoring:
+    driver: bridge
+
+services:
+  init_prometheus:
+    image: prom/prometheus:latest
+    container_name: prometheus_prestart
+    user: root
+    entrypoint:
+      - /bin/sh
+      - -c
+      - |
+        chown -R 65534:65534 /prometheus
+    volumes:
+      - ./data/prometheus/:/prometheus/
+  prometheus:
+    image: prom/prometheus:latest
+    container_name: prometheus
+    restart: unless-stopped
+    volumes:
+      - ./configs/prometheus.yml:/etc/prometheus/prometheus.yml
+      - ./data/prometheus/:/prometheus/
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+      - '--storage.tsdb.path=/prometheus/'
+      - '--web.console.libraries=/etc/prometheus/console_libraries'
+      - '--web.console.templates=/etc/prometheus/consoles'
+      - '--web.enable-lifecycle'
+    expose:
+      - 9090
+    ports:
+      - "9090:9090"
+    networks:
+      - monitoring
+    depends_on:
+      - init_prometheus
+  grafana:
+    image: grafana/grafana
+    container_name: grafana
```

Comment:
Same as prometheus container name.

Reply:
Discussed with Quentin and the docker-compose will be removed and replaced with a Terraform deployment with all three imported resources.